Plugins/upenn.cbica.captk.brain.phiestimator/files.cmake

```diff
@@ -16,7 +16,7 @@ set(MOC_H_FILES
 # the actual plugin. For example, the icon that is typically displayed in the
 # plugin view menu at the top of the application window.
 set(CACHED_RESOURCE_FILES
-  resources/icon.svg
+  resources/ExampleIcon.svg
```

Comment:
This and all the PHIEstimator changes are about the icon. It will need to be renamed to "icon" in the future, not sure why I went with that name

Reply:
ok